django_fido/management/commands/download_authenticator_metadata.py

```diff
@@ -57,7 +113,12 @@ def handle(self, **options):
         except TypeError:
             raise CommandError('access_token setting must be specified for this command to work.')
 
-        metadata = _get_metadata()
+        metadata, alg = _get_metadata()
+        # Convert alg name to corresponding hashing algorithm
+        if alg[0:2].upper() in ('HS', 'RS', 'ES'):
+            alg = 'SHA' + alg[2:]
```

Comment:
I'd rather have a mapping for algoritms.

Reply:
I agree with the mapping.

Reply:
`ALG_MAPPING` is not very descriptive. What about `HASH_ALG_MAPPING`?